wellcomeml/viz/elements.py

```diff
@@ -0,0 +1,113 @@
+from collections import defaultdict
+
+import numpy as np
+from bokeh.plotting import figure, show
+from bokeh.io import output_notebook, output_file, reset_output
+
+from wellcomeml.viz.palettes import WellcomeBackground
+from wellcomeml.viz.colors import NAMED_COLORS_DICT
+
+
+def plot_heatmap(co_occurrence,
+                 concept_order=None,
+                 file="heatmap.html",
+                 notebook=True,
+                 title="",
+                 notebook_url='localhost:8888',
+                 color='Blue Lagoon',
+                 metadata_to_display=[]):
```

Comment:
Noticed that there is a list as a default parameter, is this a deliberate choice? If not, maybe consider using None as the default for metadata_to_display as it should still evaluate correctly with the if statement within the function. In case anyone hasn't come across this before, this will help avoid any [unexpected mutability behaviours](https://docs.python-guide.org/writing/gotchas/).